src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/WindowsIdentity.cs

```diff
@@ -672,11 +673,29 @@ public static T RunImpersonated<T>(SafeAccessTokenHandle safeAccessTokenHandle,
             if (func == null)
                 throw new ArgumentNullException(nameof(func));
 
-            T result = default(T);
+            T result = default;
             RunImpersonatedInternal(safeAccessTokenHandle, () => result = func());
             return result;
         }
 
+        /// <summary>
+        /// Runs the specified asynchronous action as the impersonated Windows identity
```

Comment:
Thank you for documenting these new public APIs. We will automatically port them to MS Docs when its appropriate.

Reply:
Are they ok?Let me know if I need to change something, I cannot use nested generics in comment

Reply:
I think they are ok. I found [this StackOverflow question](https://stackoverflow.com/questions/5030091/reference-to-generic-type-in-xml-code-comment), and it seems you did the right thing.

Once we port them, the dotnet-api-docs review process will ensure the hyperlinks work properly in MS Docs, so we'll take care of them when the time comes.

Reply:
Presumably you could verify that they produce good intellisense popups in VS (ie., paste something similar into a test project in VS and see what intellisense shows). Don't know whether dotnet-api-docs has more stringent requirements, but that's one way to test something at least.

Reply:
Did a check and method and input parameter comments is ok...show return value seem not supported yet https://github.com/dotnet/roslyn/issues/31618